kevm-pyk/pyproject.toml

```diff
@@ -64,4 +64,3 @@ exclude = "src/kevm_pyk/kproj"
 
 [tool.mypy]
 disallow_untyped_defs = true
-exclude = "src/kevm_pyk/kproj/*"
```

Comment:
The code quality check issues are caused by this deletion.

Reply:
Oh, I see, I don't remember doing that...